lib/email_manager.js

```diff
@@ -61,6 +91,7 @@ exports.start = function (config_manager) {
       return transport.sendMail(email, function (err) {
         if (err) {
           // nodemailer errors are missing .message property
+          console.log(err)
```

Comment:
[`console.error` does exactly the same as `console.log` does, expect for writing to stderr instead of stdout](https://nodejs.org/api/console.html#console_console_error_data). These are the so called [standard streams](https://en.wikipedia.org/wiki/Standard_streams).

Consistently writing errors to stderr makes it possible for other programs to separate output from errors, so this is simply a matter of semantics.